src/carousel.tsx

```diff
@@ -101,6 +102,19 @@ const arrowButtonStyle = (theme: Theme) => ({
     },
 });
 
+const arrowButtonSeparation = (containerType: ContainerType, isLargeDesktop: boolean, sideMargin: number) => {
```

Comment:
Yes, I've decided to leave the balls the same (depending on the screen size) to let them feel the same (It would be weird to see some balls with different styles at the same time.
IMHO, we should not change styles of the bullets depending on the column.